src/components/EntitlementsAgreementsList/EntitlementAgreementsList.js

```diff
@@ -86,10 +108,13 @@ EntitlementAgreementsList.defaultProps = {
 };
 
 EntitlementAgreementsList.propTypes = {
-  entitlements: PropTypes.arrayOf(PropTypes.object),
+  contentData: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Same comment as above. We shouldnt be changing the API unless absolutely necessary. Also `contentData` is a prop that is speicifically used across FOLIO for the MCL. We should not be naming our app level prop the same as it might lead to confusion in the future.

Reply:
The tests also need to be fixed as a result of the prop change. Hopefully they will pass once you revert the change.